packages/injective-math/src/fp_decimal/from_str.rs

```diff
@@ -14,7 +14,7 @@ impl FromStr for FPDecimal {
     /// This never performs any kind of rounding.
     /// More than 18 fractional digits, even zeros, result in an error.
     fn from_str(input: &str) -> Result<Self, Self::Err> {
-        let sign = if input.starts_with('-') { 0 } else { 1 };
+        let sign = i8::from(!input.starts_with('-'));
```

Comment:
I think it's easier to understand the way it was before.

Reply:
Reverted. This was a suggestion from clippy v1.66.0. Interestingly, it's gone in v1.67.1.